content/en/tracing/legacy_app_analytics/_index.md

```diff
@@ -470,14 +474,14 @@ span->SetTag(datadog::tags::analytics_event, 0.5);
 
 ## Span filtering
 
-An [Analyzed Span][2] represents the top [span][3] for a [service][4], including its metadata. Once enabled, Analyzed Spans are sent at 100% throughput by default. For example, a Java service with 100 requests will generate 100 Analyzed Spans from its `servlet.request` spans, as each `servlet.request` span generates an Analyzed Span. [Filtering Analyzed Spans][5] has the benefit of reducing the number of billable Analyzed Spans and has no effect on [trace][6] sampling. Once a service has been filtered lower than 100%, the Analyzed Span generated metrics Total Errors and Total Requests are upscaled to display an estimate by default, and you have the option to display the filtered value.
+An [Indexed span][2] represents the top [span][3] for a [service][4], including its metadata. Once enabled, Indexed Spans are sent at 100% throughput by default. For example, a Java service with 100 requests will generate 100 Indexed Spans from its `servlet.request` spans, as each `servlet.request` span generates an Indexed span. [Filtering Indexed Spans][5] has the benefit of reducing the number of billable Indexed Spans and has no effect on [trace][6] sampling. Once a service has been filtered lower than 100%, the Indexed span generated metrics Total Errors and Total Requests are upscaled to display an estimate by default, and you have the option to display the filtered value.
```

Comment:
```suggestion
An [Indexed Span][2] represents the top [span][3] for a [service][4], including its metadata. Once tracing is enabled, 100% of Indexed Spans are sent by default. For example, a Java service with 100 requests will generate 100 Indexed Spans from its `servlet.request` spans, as each `servlet.request` span generates an Indexed Span. You can reduce the number of billable indexed Spans by [Filtering Indexed Spans][5], without affecting [trace][6] sampling. If a service has been filtered lower than 100%, the Indexed Span-generated metrics Total Errors and Total Requests are adjusted to display an estimate by default, and you have the option to display the filtered value instead.
```

Reply:
Oh this is now just wrong hang on will update.  Find replace did not do a good job on this one :(